src/main/resources/messages/pyrightls.properties

```diff
@@ -33,6 +33,9 @@ configurations.workspaceFolders.label = Workspace folders:
 configurations.workspaceFolders.projectBase = Project base directories
 configurations.workspaceFolders.sourceRoots = Source roots
 
+configurations.targetedFileExtensions.label = Targeted file extensions:
+configurations.targetedFileExtensions.comment = List of extensions of files on which the language server should be run.
```

Comment:
Could clarify the description and format here; something like: "List of `|`-separated extensions ..." and give an example ("... should be run. E.g. `py|pyi`")